docs/FRSKY_R9.md

```diff
@@ -70,18 +84,84 @@ The wiring is similar to the ExpressLRS docs found in the link below. Different
 The R9MX receiver can also be used as mLRS Tx module. Due to hardware restrictions it is somewhat limited but can be a great choice for special applications. 
 
 The module can be configured via CLI only. The CLI shares the same port used for serial/Mavlink communication. Switching between CLI and serial port is done via the bind button:
-- Powering up the R9MX without holding down the bind button will boot in "Serial" mode. To enter CLI mode, press the bind button while powering up. The device then runs in CLI mode, and the Rx and Tx can be configured as usual via CLI commands.
+- Powering up the R9MX without holding down the bind button will boot in "Serial" mode. To enter CLI mode, press the bind button while powering up or shortly after powering up if using the ELRS bootloader. The device then runs in CLI mode, and the Rx and Tx can be configured as usual via CLI commands.
+
+The wiring is similar to Rx and ExpressLRS, with the exception that the "Inverted SPort" pin (original naming) is available as input port.
+
 
-The wiring is similar to Rx and ExpressLRS, with the exception that the "Inverted SPort" pin (orginal naming) is available as input port.
+## R9MM Rx Module ##
```

Comment:
pl write ## R9MM Receiver ##
(we try to settle for "Tx module" and "receiver" throughout the docs)